core/src/main/java/org/elasticsearch/cluster/routing/IndexShardRoutingTable.java

```diff
@@ -330,6 +331,35 @@ public ShardIterator onlyNodeActiveInitializingShardsIt(String nodeId) {
         return new PlainShardIterator(shardId, ordered);
     }
 
+    /**
+     * Returns shards based on nodeAttributes given  such as node name , node attribute, node IP
+     * Supports node specifications in cluster API
+     *
+     * @param nodeAttribute
+     * @param discoveryNodes
+     * @return a {@link List} of shards
```

Comment:
{@link List} is wrong here right? just remove the @returns it's not helpful here since you already documented what it returns